Lib/fontTools/feaLib/lexer.py

```diff
@@ -45,13 +47,13 @@ def __init__(self, text, filename):
     def __iter__(self):
         return self
 
-    def next(self):  # Python 2
-        return self.__next__()
+    def next(self, comments = False):  # Python 2
+        return self.__next__(comments)
 
-    def __next__(self):  # Python 3
+    def __next__(self, comments = False):  # Python 3
         while True:
             token_type, token, location = self.next_()
-            if token_type not in {Lexer.COMMENT, Lexer.NEWLINE}:
+            if token_type not in (Lexer._OMITS if comments else Lexer._OMITC):
```

Comment:
I chose the route I did for speed. But I'm happy to go slow and pretty if you like.